src/config/passport.js

```diff
@@ -1,186 +1,198 @@
-var mongoose = require('mongoose'),
-    LocalStrategy = require('passport-local').Strategy,
-    TwitterStrategy = require('passport-twitter').Strategy,
-    FacebookStrategy = require('passport-facebook').Strategy,
-    GitHubStrategy = require('passport-github').Strategy,
-    GoogleStrategy = require('passport-google-oauth').OAuth2Strategy,
-    User = mongoose.model('User'),
-    config = require('./config');
+import mongoose from 'mongoose';
+import Local from 'passport-local';
```

Comment:
how about import { Strategy as localStrategy }

Reply:
I tried to do this, but I ran into errors. I think it's because of how the files are exported in the package. Should I bring this to you?